test/nginx_aaad_proxy_test.py

```diff
@@ -154,6 +154,47 @@ def main(args):
         test_proxy_user_valid_permissions_read(app_url, headers)
         test_proxy_user_valid_permissions_delete(app_url, headers, json_data)
 
+    chronos_common_access_url = ['http://chronos.dev.roger.dal.moz.com/',
```

Comment:
Is it possible to not add the roger specific details and use local cluster endpoints or maybe mock the endpoint?